### PR TITLE
Abdul e2e

### PR DESCRIPTION
**Pull request by abdul-qadir**

### CODE REVIEW

File: test/e2e/report/0003003a-004f-00a0-00dd-00ef000d0082.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"description":"register the user|registration","passed":true,"os":"MAC","browser":{"name":"chrome","version":"44.0.2403.125"},"message":"Passed.","screenShotFile":"0003003a-004f-00a0-00dd-00ef000d0082.png"}
```
Comment (vietnogi) [inline test/e2e/report/0003003a-004f-00a0-00dd-00ef000d0082.json]: should we git ignore the report folder?

**Comment by vietnogi** [comment]
What are the steps to run the tests?

File: test/e2e/report/0003003a-004f-00a0-00dd-00ef000d0082.json (hunk 1 above)
Comment (abdul-qadir) [inline test/e2e/report/0003003a-004f-00a0-00dd-00ef000d0082.json]: yeah..we can. I just kept it for understanding for first time devs.

Comment (abdul-qadir) [inline test/e2e/report/0003003a-004f-00a0-00dd-00ef000d0082.json]: @vietnogi It was added in README.md but it looks that it was overwritten. Please check this for protractorjs setup and run steps.
https://docs.google.com/document/d/1pulWegGxovvSiD4xzlASLWPZkCyy9ajXgaZjkwOkUXo/edit#heading=h.6o2u0gg3sugu

**Comment by vietnogi** [comment]
Currently, we have all of our tasks in gulp, including unit tests and e2e testings.  Would it be a good idea to run the e2e testing via GULP?  Would it be a good idea to run e2e in travis?

File: test/e2e/report/0003003a-004f-00a0-00dd-00ef000d0082.json (hunk 1 above)
Comment (abdul-qadir) [inline test/e2e/report/0003003a-004f-00a0-00dd-00ef000d0082.json]: @vietnogi  We can use gulp to run e2e tesitng. Are we not using jenkins and using travis only?

**Pull request closed without merging**